submodules/power_management_services_setup/main.tf

```diff
@@ -0,0 +1,108 @@
+#####################################################
+# Configure Squid client
+#####################################################
+
+resource "null_resource" "perform_proxy_client_setup" {
+
+  count = var.perform_proxy_client_setup != null ? length(var.perform_proxy_client_setup["squid_client_ips"]) : 0
+
+  connection {
+    type         = "ssh"
+    user         = "root"
```

Comment:
Don't think it's a good idea to default to root here. Could we make this configurable?

Reply:
in the image, other users (we could potentially use) are not available after the installation. And btw. all SAP installations from bottom to top are executed as root.

Reply:
Need root.